pyproject.toml

```diff
@@ -12,7 +12,7 @@ requires-python = ">=3.10"
 license = {file = "LICENSE"}
 authors = [{name = "Quantinuum"}]
 
-dependencies = ["pytket"]
+dependencies = ["phir>=0.1.5", "pytket"]
```

Comment:
Only question here is should the version be pinned to exactly what's in `requirements.txt`? To prevent any possibility of drift.

Reply:
I think, we should be more flexible in `pyproject.toml`. The pinned version is for reproducible builds, dev environments, etc.

The `phir` project can add more types/fields, but that may not necessarily require changes to our phir generation.